<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Step3</class>
 <widget class="QWidget" name="Step3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please, provide the sum:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inputSumLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#inputSumLabel { font-size: 20pt; }</string>
   </property>
   <property name="text">
    <string>0  UAH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="num1Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num2Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num3Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num4Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num5Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num6Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num7Button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num8Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num9Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num0Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string> &lt; Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#errorLabel {color: red;}</string>
   </property>
   <property name="text">
    <string>Insufficient funds for current transfer!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="invalidLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#invalidLabel {color: red;}</string>
   </property>
   <property name="text">
    <string>Sum is not valid!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
